### ISSUE TITLE
[BUG] Recent update to official repository breaks old projects

### ISSUE THREAD

**Issue by anatawa12** [vrchat-community/creator-companion#225]
**Describe the bug**
When we tried to resolve a project with vrcsdk 3.1.9 as a locked dependencies, vpm-resolver cannot resolve that.

Recent update to the official repository removes packages. That's too bad for package managers.

**To Reproduce**
1. Create project with VRCSDK 3.1.9
2. remove com.vrchat.base folder
3. open project on unity
4. VPM Resolver shows error

**Expected behavior**
Any packages MUST NOT be removed from repository for compatibility. Markibg as yanked is much better way to not recommend to use the package.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version vpm resolver 0.1.17

**Additional context**

issue about yanking is #204

**Comment by momo-the-monster** [comment]
@anatawa12 - my apologies, we had to emergency-migrate our packages this morning and only had time to set up the newest versions. We've now restored all the versions back to 3.0.9 - though you may need to delete the vrc-official.json listing by hand to get the newest versions (something else that should be fixed in the 2.1.0 update).

**Comment by anatawa12** [comment]
Thank you for fixing this problem. Relaunching VCC shows versions back to 3.0.9. however, because of another old bug in vpm-resolver (I found not reported yet so I'll make another issue), vpm-resolver cannot download the zip file.

```
[ERR] Couldn't download package from url https://github.com/vrchat/packages/releases/download/3.1.9/com.vrchat.base-3.1.9.zip&time=133248068902383830: NotFound
```
